deploy/cloudbuild.yaml

```diff
@@ -2,18 +2,21 @@ steps:
   # First, build kaniko
   - name: "gcr.io/cloud-builders/docker"
     args: ["build", "-f", "deploy/Dockerfile",
-           "-t", "gcr.io/kaniko-project/executor:${COMMIT_SHA}", "."]
```

Comment:
We would need to change the trigger in kaniko Project to set `PROJECT_ID` to "kaniko-project" along with this change?

Can you please test the trigger which used this file after adding this substitution value 
https://pantheon.corp.google.com/cloud-build/triggers/edit/0539d73e-5ccf-46e7-af5a-6f857f99c871?project=kaniko-project

Reply:
$PROJECT_ID will automatically be the current project that the cloud build is executing in. Does cloud build trigger run in the project `kaniko-project`?

Reply:
yes!. Great thanks for confirming.